cdap-app-fabric/src/main/java/io/cdap/cdap/internal/capability/CapabilityApplier.java

```diff
@@ -102,6 +123,10 @@
     this.namespaceAdmin = namespaceAdmin;
     this.metadataSearchClient = new MetadataSearchClient(discoveryClient);
     this.artifactRepository = artifactRepository;
+    this.remoteClient = new RemoteClient(discoveryClient,
```

Comment:
Why are we using remote client to take to app-fabric? This class runs inside app-fabric, right?

Reply:
Removed remote client. Inject artifact repository instead.